spec/fixtures/files/datahashes/core/acquisition1.json

```diff
@@ -24,15 +24,15 @@
     "approvalGroup": ";collection committee;executive committee",
     "approvalIndividual": "malik;;rudelyt",
     "approvalStatus": "not approved;not required;approved",
-    "approvalDate": "approvalDate;2020-09-25;",
+    "approvalDate": "2020-09-23;2020-09-25;",
     "approvalNote": "Hands;;Loving",
     "acquisitionNote": "Going home",
     "acquisitionProvisos": "Pictures on the wall",
-    "acquisitionFundingCurrency": "Danish Krone;;Euro",
-    "acquisitionFundingValue": "250;85;",
-    "acquisitionFundingSourcePerson": "Disturb;",
-    "acquisitionFundingSourceOrganization": "But Ohh;",
-    "acquisitionFundingSourceProvisos": "Krone;Clone;",
```

Comment:
Had to change the order of these in the CollectionSpace record/XML because the mapper maps multi-authority fields in order that the columns appear in CSV, so rows with Organization values in acquisitionFundingSource will always come after rows with Person values in acquisitionFundingSource.

Also in acquisitionFundingSourcePerson & acquisitionFundingSourceOrganization, you were sending through values for FOUR rows: "Disturb;{blank Person};But Ohh;{blank Organization}"